README.md

```diff
@@ -47,3 +47,12 @@ For zsh shell, instead use:
 For fish shell, instead use:
 
 `. ~/.asdf/plugins/java/set-java-home.fish`
+
+Note that the location of the scripts will be different if you installed asdf with a package manager.
+For example, you you have installed asdf using brew, the scripts will be located in
+
+`/usr/local/opt/asdf/plugins/java/`
```

Comment:
Should we use the output of `brew --prefix asdf` instead? The path is different if Homebrew was installed in a different location:
https://github.com/Homebrew/brew/blob/2100335441a7068abb5a08be1e01086b44617ae5/docs/Installation.md#alternative-installs

```suggestion
`$(brew --prefix asdf)/plugins/java/`
```

Reply:
Good call!